frontend/mobilityai_react/App/Screens/PatientBarGraphs.js

```diff
@@ -43,24 +56,35 @@ export default class BarGraph extends Component {
                         fill: 'grey',
                         fontSize: 10,
                     }}
-                    // numberOfTicks={this.props.yLabels.length}
                     style={{ height: 200 }}
                 />
                 <ScrollView horizontal={true} style={styles.bargraph}>
-                    <View style={{ width: this.state.graphWidth}}>
-                        <BarChart
-                            style={{ height: 200 }}
-                            data={this.props.data}
-                            svg={{ fill }}
-                            contentInset={{ top: 30, bottom: 30 }}
-                            yMin={0}
-                            yMax={this.props.yLabels.length}
-                        >
+                    <View style={{ width: 500 }}>
+                        {this.props.singleBarView 
+                            ? <BarChart
+                                style={{ height: 200 }}
```

Comment:
We use this style height a lot, we should factor it out into the styles object at the bottom.